packages/ember-routing/lib/location/history_location.js

```diff
@@ -101,7 +101,8 @@ export default EmberObject.extend({
     // remove baseURL and rootURL from start of path
     let url = path
       .replace(new RegExp(`^${baseURL}(?=/|$)`), '')
-      .replace(new RegExp(`^${rootURL}(?=/|$)`), '');
+      .replace(new RegExp(`^${rootURL}(?=/|$)`), '')
+      .replace(/\/\//g,'/'); // remove extra slashes
```

Comment:
Should this be anchored? If not, can you add a test showing that something like `/foo//bar/` is actually supported?